dapp/package.json

```
@@ -6,8 +6,8 @@
   "scripts": {
     "start": "node src/app.js",
     "start-local": "node -r dotenv/config src/app.js",
-    "test": "jest --setupFiles dotenv/config --silent=false",
-    "ctest": "jest --setupFiles dotenv/config --silent=true --coverage",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "ctest": "node --experimental-vm-modules node_modules/jest/bin/jest.js --silent=true --coverage",
```

we may want to directly invoke `jest` instead of `node`

```suggestion
    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
    "ctest": "NODE_OPTIONS=--experimental-vm-modules jest --silent=true --coverage",
```